include/dt-bindings/adc/adc.h

```diff
@@ -0,0 +1,31 @@
+/*
+ * Copyright (c) 2021 Linaro Limited
+ *
+ * SPDX-License-Identifier: Apache-2.0
+ */
+#ifndef ZEPHYR_INCLUDE_DT_BINDINGS_ADC_ADC_H_
+#define ZEPHYR_INCLUDE_DT_BINDINGS_ADC_ADC_H_
+
+/*
+ * Provide the BIT_MASK() macro for when this file is included from
```

Comment:
is there a header we can include to get this?

Reply:
BIT_MASK is defined in sys/util.h, which includes additional headers and defines C prototypes which cause devicetree errors.